docs/source/models/supported_models.rst

```diff
@@ -51,6 +51,10 @@ Alongside each architecture, we include some popular models that use it.
     - Falcon
     - :code:`tiiuae/falcon-7b`, :code:`tiiuae/falcon-40b`, :code:`tiiuae/falcon-rw-7b`, etc.
     -
+  * - :code:`FuyuForCausalLM`
```

Comment:
We have recently split out the list of supported VLMs from the main list (#6167). Please update accordingly.